test_conformance/allocations/main.cpp

```diff
@@ -24,8 +24,10 @@
 
 typedef long long unsigned llu;
 
+#define REDUCTION_PERCENTAGE_DEFAULT 50
```

Comment:
I'm not necessarily opposed to this, but it does change the default behavior of the test pretty significantly, since a "reduction percentage" less than 100% also affects the number of work-items executed.

Should we just change the number of work-items executed unconditionally?  I'd feel more comfortable if this were just changing the max allocation percentage.